app/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CheckYourAnswersController.scala

```diff
@@ -123,10 +130,16 @@ class CheckYourAnswersController @Inject()(journeyService: JourneyService,
                 _ <- storageService.storeRegistrationStatus(journeyId, RegistrationNotCalled)
                 _ <- auditService.auditJourney(journeyId, authInternalId)
               } yield Redirect(routes.JourneyRedirectController.redirectToContinueUrl(journeyId))
-              case _ => for {
-                _ <- storageService.storeIdentifiersMatch(journeyId, identifiersMatch = false)
-                _ <- auditService.auditJourney(journeyId, authInternalId)
-              } yield Redirect(errorRoutes.CtutrMismatchController.show(journeyId))
+              case DetailsMismatch if optCtutr.isEmpty =>
+                for {
+                  _ <- storageService.storeIdentifiersMatch(journeyId, identifiersMatch = false)
+                  _ <- auditService.auditJourney(journeyId, authInternalId)
```

Comment:
We need to store the BV status before we audit here, otherwise it will be audited as not requested